src/quantum/azext_quantum/tests/latest/utils.py

```diff
@@ -53,3 +53,22 @@ def all_providers_are_in_capabilities(provider_sku_string, capabilities_string):
         if provider not in capabilities_string:
             return False
     return True
+
+# import pytest
```

Comment:
If you merge with this, please open an item to track fixing this properly.

Reply:
A work item has been created.  The unit tests could be more robust if this pytest fixture could be used.